packages/uniforms-bootstrap5/src/ListAddField.tsx

```diff
@@ -0,0 +1,55 @@
+import classnames from 'classnames';
+import cloneDeep from 'lodash/cloneDeep';
+import React, { ReactNode } from 'react';
+import {
+  connectField,
+  filterDOMProps,
+  HTMLFieldProps,
+  joinName,
+  useField,
+} from 'uniforms';
+
+export type ListAddFieldProps = HTMLFieldProps<
+  unknown,
+  HTMLDivElement,
+  { addIcon?: ReactNode; initialCount?: number }
+>;
+
+function ListAdd({
+  addIcon,
+  className,
+  disabled,
+  name,
+  readOnly,
+  value,
+  ...props
+}: ListAddFieldProps) {
+  const nameParts = joinName(null, name);
+  const parentName = joinName(nameParts.slice(0, -1));
+  const parent = useField<{ maxCount?: number }, unknown[]>(
+    parentName,
+    {},
+    { absoluteName: true },
+  )[0];
+
+  const limitNotReached =
+    !disabled && !(parent.maxCount! <= parent.value!.length);
+
+  return (
+    <div
+      {...filterDOMProps(props)}
+      className={classnames('badge rounded-pill float-end', className)}
+      onClick={() => {
+        if (limitNotReached && !readOnly) {
+          parent.onChange(parent.value!.concat([cloneDeep(value)]));
+        }
+      }}
+    >
+      {addIcon}
+    </div>
+  );
+}
+
+ListAdd.defaultProps = { addIcon: <i className="octicon octicon-plus" /> };
```

Comment:
I'm just thinking, whether we should assume that Octicons are the default. Maybe we should switch to `+`/`-` or other unicode characters, just like in `uniforms-unstyled`? 

@Monteth, @wadamek65

Reply:
Personally, I would opt for a nicer look out of the box for themes other than `uniforms-unstyled`, as it's a better selling point for new users to provide polished form by just writing a schema, than telling users could easily configure them.

Reply:
@michael-vasyliv: what do you think? I'm not using Bootstrap nowadays and your experience here may be crucial.

Reply:
@radekmie I guess it's fine to have `Octicons` anyway I don't use `Octicons` in my project. However it's easy to change, if we add smth like `unicode character` then we will have to create a component wrapper where we will use other characters (and component wrapper for AutoForm to replace it in `switch ... case`) (in this case will do more work).
It's much easy to have smth like this:
```
.octicon-plus:before {
    content: "\271A";
}
```

Reply:
OK, let's leave it as it is.